scripts/merge_tract_cat.py

```diff
@@ -184,33 +184,52 @@ def load_patch(butler_or_repo, tract, patch,
             print("  No good isPrimary entries for tract %d, patch %s" % (tract, patch))
         return ref_table
 
+    flux_field_names_per_schema_version = {
+        1: {'psf_flux': 'base_PsfFlux_flux', 'psf_flux_err': 'base_PsfFlux_fluxSigma',
+              'modelfit_flux': 'modelfit_CModel_flux', 'modelfit_flux_err': 'modelfit_CModel_fluxSigma'},
+        2: {'psf_flux': 'base_PsfFlux_flux', 'psf_flux_err': 'base_PsfFlux_fluxErr',
+              'modelfit_flux': 'modelfit_CModel_flux', 'modelfit_flux_err': 'modelfit_CModel_fluxErr'},
+        3: {'psf_flux': 'base_PsfFlux_instFlux', 'psf_flux_err': 'base_PsfFlux_instFluxErr',
+              'modelfit_flux': 'modelfit_CModel_instFlux', 'modelfit_flux_err': 'modelfit_CModel_instFluxErr'},
```

Comment:
Where is `psf_flux_err` coming from? I think it, and the modelFit one should be `fluxErr`.